### PR TITLE
Clarify how to use id constraints in XPath

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/refguide/modeling/xpath/xpath-constraints/_index.md [hunk 1]
```diff
@@ -17,16 +17,20 @@ The following example shows how you should perform an XPath query in Studio Pro.
...
+## 2 Multiple Constraints
 
-## 2 Examples
+Multiple constraints can be added to a single query for all queries except where you are querying on the `id` (the unique identifier) of the object.
 
-{{% alert color="info" %}}
+If you need to query on `id` (for example `[id = $currentuser]`) as part of multiple constraints, you can create an `and` constraint by using the first format shown in the [Constraint One `and` Constraint Two](#and) section: `[id = …][{additional constraint}]`.
 
+{{% alert color="info" %}}
 For all the XPath constraints examples given for Studio Pro, it is assumed that you have already selected the relevant entity as the context of the XPath constraints before you type in the constraints. For all the examples given in Java environment, whole XPath queries are presented, including the double slashes (`//`) and the entity name.
```
Comment (torben-graversen-mendix) [inline content/en/docs/refguide/modeling/xpath/xpath-constraints/_index.md]: This section feels duplicate to the "warning" box at the top. Which I also wonder why is a "warning", there's no risk?
How about this box here just says:
"All the Studio Pro examples below assume that an entity `Sales.Customer` was selected for retrieval."

Comment (torben-graversen-mendix) [reply content/en/docs/refguide/modeling/xpath/xpath-constraints/_index.md]: Also, should the introduction start with just the basics about what an XPath Constraint is? That seems like the most important message, and starting the document with a warning seems intimidating.
I think the first paragraph could go before the "warning" box.
1. A constraint can be added to any XPath .... etc.
2. The syntax of XPath constraints differ between ....
3. The following example shows....

Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/xpath/xpath-constraints/_index.md]: @torben-graversen-mendix 
Great feedback, thanks.
I'll look into that. I'm hoping to use the introduction of the XPath builder to review all the XPath documentation and make it clearer and less intimidating - to match the benefits of the builder.

**Comment by MarkvanMents** [comment]
Changes agreed over Slack.
Will do further work on a new PR about XPath builder.

**Pull request merged**